src/platform/macos/events_loop.rs

```diff
@@ -680,6 +680,23 @@ pub fn to_virtual_key_code(code: c_ushort) -> Option<events::VirtualKeyCode> {
     })
 }
 
+pub fn to_virtual_key_code_ex(
```

Comment:
The suffix `ex` seems like a Windows naming convention to me, and feels a bit out of place in macOS code. Either way, a more explicit/specific name would be nice, since this function is exclusively for function keys. It would also be good to have a comment briefly explaining why a separate function is needed, paired with a link to the [relevant docs](https://developer.apple.com/documentation/appkit/1535851-function_key_unicodes?language=objc).